src/tests/nativeaot/SmokeTests/HelloWasm/WasmDebuggingTestLauncher.cmd

```diff
@@ -0,0 +1,7 @@
+echo ====== WasmDebuggingTestLauncher.cmd ======
```

Comment:
As I understand it, we are not running the automatic testing in CI for windows due to a lack of `lldb`, so is this for manual running?

Reply:
Yep.